src/daemon/server.py

```diff
@@ -871,9 +895,10 @@ async def async_run_daemon(root_path):
     create_server_for_daemon(root_path)
     ws_server = WebSocketServer(root_path, ca_crt_path, ca_key_path, crt_path, key_path)
     await ws_server.start()
+    return 0
```

Comment:
Making it consistent with the return type, returning always int, not int or None.